docs/application/vstools/install.md

```diff
@@ -57,15 +57,15 @@ Tizen Emulator for Visual Studio has the same requirements as the emulator in Ti
 <a name="install"></a>
 ## Visual Studio Tools for Tizen installation
 
-To use the Tizen SDK tools, you must install the Visual Studio Tools for Tizen extension and the Baseline SDK.
+To use Tizen SDK tools, you must install Visual Studio Tools for Tizen extension and Baseline SDK.
 
 > [!NOTE] 
 > If Tizen Studio is already installed on your computer, simply [set the Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
```

Comment:
For nouns such as tool names, it is recommended to not use "The".
```suggestion
> If Tizen Studio is already installed on your computer, simply [set Tizen Baseline SDK path](#set-tizen-baseline-sdk) to the existing SDK instead of reinstalling it.
```